src/renderer/theme.store.ts

```diff
@@ -29,18 +29,8 @@ import type { SelectOption } from "./components/select";
 
 export type ThemeId = string;
 
-export enum MonacoTheme {
-  DARK = "clouds-midnight",
-  LIGHT = "vs"
-}
-
-export enum ThemeType {
-  DARK = "dark",
-  LIGHT = "light",
-}
```

Comment:
Why remove the enums?

Reply:
Those are used only internally and redundant.